doc/user_doc/vic_installation/plugin_vc_sftp.md

```diff
@@ -6,10 +6,11 @@ If your vCenter Server instance runs on a Windows system that has access to an S
 
 - You deployed at least one virtual container host to a vCenter Server instance that runs on Windows.
 - Your vCenter Server instance is running on a Windows system that has an SFTP service.
+- You must use a Windows system to run the script to install the plug-in on a vCenter Server that runs on Windows. If you used a Linux or Mac OS system to deploy the virtual container host, download and unpack the vSphere Integrated Containers Engine package on a Windows system.
```

Comment:
@jooskim Is this correct? I can see that there's only a `.bat` file in the installer/vCenterForWindows directory however it would seem that the type of the target vCenter is the important criteria, not what OS we're running the script on.